index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
What is this file for? There is already a `index.html` file in `src`

Reply:
I have no idea why but my extremely old commit from last month kept on appearing. It shouldn't appear again though because I've removed it and the very old commits from my history